vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -147,110 +100,143 @@ chmod +x ./push-images.sh
   </Step>
   <Step>
 
-Run `download-images.sh` to download all images locally:
+Run `download-images.sh` to pull all images locally and create a tarball of all of the images:
 
 ```bash title="Download images"
 ./download-images.sh --image-list images.txt
 ```
 
-:::tip
-This creates a tarball with all the images and push them to your private registry.
+:::warning
+Review the output to confirm all images were pulled succesfully and packaged in the tarball. 
 :::
 
   </Step>
   <Step>
 
-Run `push-images.sh` to push all downloaded images to your private registry:
+Run `push-images.sh` to push all the images to your private image registry:
 
 ```bash title="Push images to private registry"
-./push-images.sh --registry ${REGISTRY}
+
+./push-images.sh --registry ${REGISTRY} 
 ```
 
 :::info
 vCluster prepends the image registry to all images used by vCluster, such as syncer and Kubernetes. For example, `registry.k8s.io/kube-apiserver:v1.30.2` becomes `my-private-registry:5000/vcluster/kube-apiserver:v1.30.2`.
```

Comment:
becomes `my-private-registry:5000/loft-sh/kube-apiserver:v1.30.2` here.

Reply:
Can you help me clarify what image gets uploaded to the private registry? It's not a direct push?